src/test/Testcafe/page-objects/landingPage.js

```diff
@@ -0,0 +1,10 @@
+import { Selector, t } from 'testcafe';
+
+export default class landingPage () { 
```

Comment:
`LandingPage`
Classnames should start with a capitalized letter.

Reply:
Fixed in new commit